src/main/java/org/fenixedu/academic/domain/dml/DynamicFieldDescriptor.java

```diff
@@ -37,6 +38,7 @@ protected void init(final String domainObjectClass, final String code, final Loc
         setName(name);
         setFieldValueClassName(fieldValueClass);
         setRequired(required);
+        DynamicFieldTag.setOrCreateDefaultTag(this);
```

Comment:
it would be more clean and consistent it this would be inverted, something like: `setTag(DynamicFieldTag.getOrCreateDefault())`